src/zapv2/__init__.py

```diff
@@ -61,8 +60,7 @@ class ZAPv2(object):
     # base OTHER api url
     base_other = 'http://zap/OTHER/'
 
-    def __init__(self, proxies={'http': 'http://127.0.0.1:8080',
-        'https': 'http://127.0.0.1:8080'}, apikey=None):
+    def __init__(self, proxies=None, apikey=None):
```

Comment:
A small python pitfall: it's dangerous to keep default values as a mutable objects: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html